src/generators/zip/index.ts

```diff
@@ -0,0 +1,9 @@
+import {Chance} from 'chance';
+
+/**
+ * Returns a random valid zip, with the ability to specify ZIP+4 format
+ */
+const createZipGenerator = (chance: Chance.Chance) => (opts?: Chance.Options | undefined): string =>
```

Comment:
```typescript
opts?: Chance.Options
```

@jason-h-hu this isn't correct. This is a generic options struct: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/612590208c4c6b605b97332028a0ace340a287f3/types/chance/index.d.ts#L199-L201. It basically allows everything. We want to only allow `{plusfour?: boolean}`.

... Ah, unfortunately, `zip()` isn't even typed properly so there's no specific type we can import: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/612590208c4c6b605b97332028a0ace340a287f3/types/chance/index.d.ts#L115.

Your call, but I would:
- hardcode `opts?: {plusfour?: boolean}` here for now
- open a PR in DefinitelyTyped adding the missing type